### DISCUSSION TITLE
College email is verified but keeps github keeps asking to add  email

### DISCUSSION THREAD

**Discussion by ghost** [community/community#125666] [GitHub Education]
### Hi everyone,

I am trying to apply for github education as a Teacher. I teach in Kent Institute Australia and i can see my institute  is listed in the dropdown. However when i try to apply , it keeps messaging 
![image](https://github.com/community/community/assets/170798974/f1885973-8f3e-4689-b033-249344d84476)
Even after refreshing.

The continue button is disabled. Not sure what is the issue.

**Comment by pedrolemos4** [comment]
Have you seen this documentation? [Apply to GitHub Education as a teacher](https://docs.github.com/pt/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-teachers/apply-to-github-education-as-a-teacher)
